apps/codeforafrica/content/projects/pesacheck.md

```diff
@@ -2,41 +2,41 @@
 id: 31626677-0bc8-49cb-a2e6-1d7e5504d1fe
 name: PesaCheck
 tagLine: Africa fact-checking
-title: Africa’s largest **indigenous fact-checking** organisation
+title: >
+  Africa’s largest <span class="highlight">indigenous fact-checking</span>
+  organisation
 subtitle: PesaCheck, a pioneering verification initiative, debunks misleading
   claims with full-time fact-checkers in 15 African countries.
-description: >-
-  <p>PesaCheck is a pioneering verification initiative that is kickstarting
-  fact-checking across Africa. Initially focused on verifying the financial and
-  other statistical numbers quoted by public figures in Kenya, Tanzania and
-  Uganda, PesaCheck is now Africa’s largest with full-time fact-checkers in 15
-  countries in both east and west Africa, as well as across the Sahel.</p>
-
-
-  <p>PesaCheck fact-checks in two international languages (English and French), as well as major African languages such as Kiswahili and Amharic. Our network helps track political promises by politicians (through our Wajibisha/PromiseTracker toolkit), helps unpack budget and census data (through our PesaYetu and TaxClock platforms), and builds machine learning/artificial intelligence tools (such as DebunkBot) to help automate verification.</p>
-category: Projects
+tag: Initiatives
 icon:
   src: https://res.cloudinary.com/code-for-africa/image/upload/v1652431239/codeforafrica/icons/Type_PesaCheck_rmswvg.svg
 thumbnail:
   src: https://res.cloudinary.com/code-for-africa/image/upload/v1652705959/codeforafrica/images/Property_1_PesaCheck_iahlrh.jpg
-href: /projects/pesa-check
+href: /projects/pesacheck
```

Comment:
I guess in most (all?) of these items, we won't need internal `href` anymores since we can generate them from `slug`.